examples/ipython/langevin_integrator_check.ipynb

```diff
@@ -122,7 +122,7 @@
    "outputs": [],
    "source": [
     "for i in range(nframes):\n",
-    "    # generate the next frame (which is sim.nsteps_per_iteration timesteps)\n",
+    "    # generate the next frame (which is sim.n_steps_per_iteration timesteps)\n",
```

Comment:
problem isn't `nsteps`=>`n_steps`, it is `per_iteration`=>`per_frame`. Old (or possibly never actually used?) name for the same concept -- just used a comment, but we should make the comments match code when we catch a discrepancy!